src/circleci/rollcage/system.clj

```diff
@@ -0,0 +1,38 @@
+(ns circleci.rollcage.system
+  "Functions that introspect the underlying system."
+  (:require [clojure.string :as string]
+    [clojure.java.io :as io]
+    [clojure.edn :as edn])
+  (:import [java.net InetAddress UnknownHostException]))
+
+(defn version
```

Comment:
This doesn't do what you expect if run from an uberjar.

Here's an example of the result from one uberjar I picked that's currently running:

```clojure
user=> (some-> (clojure.java.io/resource "project.clj") slurp clojure.edn/read-string clojure.pprint/pprint)
(defproject
 clj-url
 "1.0.3"
 :description
 "a library for parsing and emitting URLs"
 :dependencies
 [[org.clojure/clojure "1.2.1"]])
```

Reply:
In the next version of Leiningen there will be metadata in the uberjar manifest that will do what you want: https://github.com/technomancy/leiningen/pull/2379